src/App.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { Route, Switch } from 'react-router';
-import { PrivateRoute } from './components/Shared/PrivateRoute';
 
+import './App.scss';
+import { PrivateRoute } from './components/Shared/PrivateRoute';
```

Comment:
Let's also make file names look like shared/private-route
So make it all lowercase and use dashes to separate